lib/tpl/dokuwiki/css/design.less

```diff
@@ -172,10 +172,63 @@ form.search {
 .dokuwiki div.breadcrumbs {
     border-top: 1px solid @ini_border;
     border-bottom: 1px solid @ini_background;
-    margin-bottom: .5em;
     font-size: 0.875em;
     clear: both;
 
+	.youarehere, .trace {
+		& {
+			display: grid;
+			grid-template-columns: min-content auto;
+			max-width: 100%;
+			overflow: hidden;
+		}
+
+		h2 {
+			display: inline;
+			font-size: 1em;
+			font-weight: normal;
+			margin: 0; padding: 0;
+		    line-height: 1.5rem;
+			white-space: nowrap;
+		}
+		ol {
+			& {
+				display: inline-flex;
+				list-style: none inside;
+				max-width: 100%;
+				overflow: hidden;
+				margin: 0; padding: 0 .5em;
+				line-height: 1.5rem;
+			}
+			li {
+				& {
+					display: inline-block;
+					margin: 0;
+					white-space: nowrap;
+					overflow: hidden;
+					width: auto;
+					text-overflow: ellipsis;
+				}
+				&:first-child,
+				&:last-child {
+					min-width: fit-content;
+				}
+				&::before {
+					content: '\00B7';
+					display: inline-block;
+					width: 1em;
+					text-align: center;
+				}
+				&:first-child::before {
+					content: none;
+				}
+			}
+		}
+	}
```

Comment:
In the dokuwiki template, my personal aim is to keep the amount of CSS as low and simple as possible and I am wondering if the same result couldn't be achieved with much less code. Basically all we do is replacing a bunch of spans with lis. So setting the li to inline should result in the same layout as before without having to use a bunch of grid and flex layouts...

Reply:
CSS is relatively low-impact in terms of performance. The biggest hit here is really the download overhead for very large files. Because of that I am not too worried about nailign down features to they work in all situations. 

Also, I have to say that I am starting to like LESS for being able to make such neat structures as just above the quoted part :-) but a side effect of that is of course that there will be deeply nestled LESS code structures now.

In this case, the added CSS also creates a new feature: this improves the handling on smaller screens, and/or with deeply nestled structures, but before the mobile view kicks in:
![image](https://github.com/user-attachments/assets/6454b8b5-80d8-44b8-84cc-064413ba4210)

I can remove this again, but I really think this is an improvement here :-)

Reply:
generally I am not concerned about the generated css, but the amount/complexity about the source. I want to keep things easy to understand and maintain.

Reply:
OK, I tried to unroll this section as far as it seems usefule. Not sure if this is more readable - but maybe it is just that I am very much used to working with deeply nestled structures... either way, good for me :-)